Example/fastlane/Matchfile

```diff
@@ -4,8 +4,9 @@ storage_mode("git")
 
 type("development") # The default type, can be: appstore, adhoc, enterprise or development
 
-# app_identifier(["tools.fastlane.app", "tools.fastlane.app2"])
-# username("user@fastlane.tools") # Your Apple Developer Portal username
+# comment out app_identifier, and username if running locally
```

Comment:
Do you want to add a note about this to the README?

Reply:
Thanks for the callout!